docs/docusaurus.config.js

```diff
@@ -27,6 +27,32 @@ module.exports = {
         alt: 'Airy Documentation',
         src: 'img/logo.svg',
       },
+      items: [
+        {
+          target: '_self',
+          label: 'Airy Core',
+          position: 'left',
+          href: 'https://airy.co/docs/airycore',
+        },
+        {
+          target: '_self',
+          label: 'Airy Enterprise',
+          position: 'left',
+          href: 'https://airy.co/docs/airyenterprise',
```

Comment:
this doesn't exist yet I think @sgarroum wrote the issue with the future in mind... :)

https://enterprise.airy.co

Reply:
perfect, was just writing to him 👍